python/ops.py

```diff
@@ -13,12 +13,37 @@
 # limitations under the License.
 """FFCC TensorFlow ops."""
 import math
-import sys
-
+import itertools
 import numpy as np
 import tensorflow.compat.v1 as tf
 
 
+
+EPS = tf.constant(1e-9, dtype=tf.float32)
+
+
+def edge_kernel():
+  """Construct a set of 3x3 kernels for edge computation."""
```

Comment:
Can you describe what these kernels look like in a short sentence?

Reply:
Added a description of what it would produce after convolution.